src/crocodilehunter.py

```diff
@@ -53,6 +53,7 @@ def __init__(self, args):
             self.project_name = args.project_name
         else:
             self.project_name = self.config['general']['default_project']
+            args.project_name = self.config['general']['default_project']
```

Comment:
I suspect you didn't mean to add a duplicate line here?

Reply:
Indeed, the line is almost the same, but this is for the case where you don't specify any project with the -p flag, in that case the program doesn't pass to watchdog.py the value of the default project in config.ini because is not present in args. 

To reproduce this we just need to start crocodile hunter using the default project in the config.ini (not using the -p flag), opening the web interface and watching the project in the upper right corner, it says Project:None and I'm suspecting the database is also not created.

Reply:
Yup that's my sloppy coding. The real fix for this is of course a refactor and tests but this seems good enough for now.